modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequestDetailPage.testcase

```diff
@@ -161,3 +161,26 @@ definition {
         }
     }
 
+    @description = "This is a test for LPS-181774. Verify that when a MDF is submitted, the activity status is Submitted and Unclaimed"
```

Comment:
Hi @marinaasant0s, I noted a typo here. Could you please replace `...when a  Mdf...` to `...when an MDF...`? Thanks!